traits/trait_types.py

```diff
@@ -361,17 +344,15 @@ def create_editor(self):
 
 
 class Str(BaseStr):
-    """ Defines a trait whose value must be a Python string using a C-level
-        fast validator.
+    """ Defines a trait that holds a str and uses a C-level fast validator.
```

Comment:
replace "a str" with "a str object"? Or keep it as "a string"? Or even "a text string"?